bin/reprocessing_routines.sh

```diff
@@ -207,3 +207,12 @@ applyFixes() {
 	fi
     done
 }
+
+# Returns prod or test, depending on the Atlas environment in which the script calling it is running
+# It is assuming that all atlasinstall_<env>s are under ${ATLAS_PROD}/sw (it will fail otherwise)
+atlas_env() {
```

Comment:
Sorry, should have made the suggestion earlier. But if you're moving away from atlas-prod maybe functions like this could be called from atlas-bash-util? https://github.com/ebi-gene-expression-group/atlas-bash-util/blob/2d9c0893de2e1fe98c67f320f74466c2d2351646/generic_routines.sh#L71

With atlas-bash-util installed this call would become:

```
atlas-bash-util atlas_env
```

Reply:
thanks @pinin4fjords, I've added that to my list!